src/applications/edu-benefits/0994/prefill-transformer.js

```diff
@@ -12,9 +12,26 @@ export function prefillTransformer(pages, formData, metadata) {
     emailAddress: formData.emailAddress,
   };
 
+  // Determine if bank information page should start in edit mode
```

Comment:
this shouldn't be here. Do we need to check this at pre-fill time? Can this not be checked in the page?